corehq/couchapps/cloudcare_apps/views/view/map.js

```diff
@@ -0,0 +1,14 @@
+function(doc){
+    if((doc.doc_type == 'Application') && doc.copy_of == null && doc.cloudcare_enabled) {
```

Comment:
I think I can safely make that change without the two views functioning any different.